src/modules/launcher/Wox.Plugin/IPlugin.cs

```diff
@@ -17,5 +17,8 @@ public interface IPlugin
 
         // Localized description
         string Description { get; }
+
+        // Plugin ID for validating the plugin.json entry (It must be static for accessing it before loading the plugin.)
+        public static string PluginID { get; }
```

Comment:
This isn't working as expected. I don't get any warning if the property is missing wen building the plugin code.

Maybe caused by this: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/static-abstracts-in-interfaces

Reply:
My research shows that making it abstract should fix it. But I don't know if we can use .Net 6 Preview or .Net 7 features yet.